docs/en/entity_statement.rst

```diff
@@ -34,6 +33,24 @@ attributes:
    * - **Claim**
      - **Description**
      - **Supported by**
+   * - **iss**
+     - See `OIDC-FED`_ Section 3.1 for further details.
+     - |spid-icon| |cieid-icon|
+   * - **sub**
+     - See `OIDC-FED`_ Section 3.1 for further details.
+     - |spid-icon| |cieid-icon|
+   * - **iat**
+     - See `OIDC-FED`_ Section 3.1 for further details.
+     - |spid-icon| |cieid-icon|
+   * - **exp**
+     - See `OIDC-FED`_ Section 3.1 for further details.
+     - |spid-icon| |cieid-icon|
+   * - **jwks**
+     - Federation JWKS of the *sub* entity. See `OIDC-FED`_ Section 3.1 for further details.
+     - |spid-icon| |cieid-icon|
+   * - **authority_hint**
```

Comment:
are we sure to also have auth hint in the ES and not only in the EC?
I think that it would be necessary only in the EC, do we agree?